### ISSUE TITLE
Text for 6 meeting materials on front page disappears with Wide Browser

### ISSUE THREAD

**Issue by unity1979** [RecoveryDharma/website#118]
On several pages (notably the front page: https://recoverydharma.org ) the text for the 6 materials (The Practice, 4NT, 8FP, DOM, CoreIntentions, Meditations) is sometimes invisible and disappears. This happens when a browser is extremely wide (Width > 900pixels). 

Notice the text is invisible here with width >900.
<img width="896" alt="image" src="https://user-images.githubusercontent.com/70724023/160705097-054912a2-9b9a-4d30-bdf1-89c27e60ee75.png">

Notice, however, the correct behavior that the text is always visible with width < 900, in the **same browser**
<img width="819" alt="image" src="https://user-images.githubusercontent.com/70724023/160705326-16335dfb-0a92-491a-9ad7-a58b098c8250.png">

Expected behavior: 
The text should always be visible. This will help with facilitators who are trying to live sharescreen for these readings.
 
Bug also because: 
if on an iPad, the text would never be revealed because we would not be able to access onHover over the image to reveal the specific text. 

Some (maybe not all) of the affected pages:
- https://recoverydharma.org
- https://recoverydharma.org/resources/meeting-materials/
- https://recoverydharma.org/get-started/
- https://recoverydharma.org/meetings/

**Comment by tom-raley** [comment]
I noticed this bug right before we launched, so I set a breakpoint at 980px screen width so that screens smaller than that should have the text visible. Unfortunately, it's hard to code for every single possible screen size. My recommendation if we're concerned about touch devices (where there is no hover effect) with large screens, then we should remove the hover effect altogether and just have the text always be visible. @Lacey-boop and @KrisRecoveryDharma let me know what you wanna do!
